gems/smithy/spec/smithy/plan_spec.rb

```diff
@@ -5,7 +5,7 @@ module Smithy
     let(:fixture) { File.expand_path('../fixtures/weather/model.json', __dir__) }
     let(:model) { JSON.load_file(fixture) }
     let(:type) { :client }
-    let(:options) { {} }
+    let(:options) { { welds: true, polishes: true } }
```

Comment:
Is there a way to invert this or default in plan initialize? New plans should not have these opt in but rather opt out.

no-welds and no-plan will set nil and not false so maybe there's a distinction there.

Reply:
Theres not a good way to invert this with Thor's boolean arguments that I could find.  I guess I could add a `--no-weld` boolean argument instead?

Reply:
Edit: Nvm - I was able to just add negative options for these instead, which I agree is preferable.